wasmsdk/README.md

```diff
@@ -183,6 +183,51 @@ updates allocation settings
 > hash: string
 
 
+### zcn.sdk.getAllocationFromAuthTicket
```

Comment:
I would suggest to name it with `getAllocationWith(authTicket)`

Reply:
fixed in 9125454